### PR TITLE
New style for TU Dresden

### PR DESCRIPTION
**Pull request by jmiba**

### CODE REVIEW

File: technische-universitat-dresden-historische-musikwissenschaft-note.csl [hunk 1]
```diff
@@ -0,0 +1,493 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style class="note" version="1.0" initialize-with="." default-locale="de-DE" xmlns="http://purl.org/net/xbiblio/csl">
+  <info>
+    <title>Technische Universität Dresden - Historische Musikwissenschaft (note)</title>
```
Comment (rmzelle) [inline technische-universitat-dresden-historische-musikwissenschaft-note.csl]: For styles fixed to a non-English locale, please include the language in the style title (i.e., `<title>Technische Universität Dresden - Historische Musikwissenschaft (note, German)</title>`)

File: technische-universitat-dresden-historische-musikwissenschaft-note.csl [hunk 2]
```diff
@@ -0,0 +1,493 @@
...
+    <link href="http://www.zotero.org/styles/foerster-geisteswissenschaft" rel="template"/>
+    <link href="https://bildungsportal.sachsen.de/opal/auth/RepositoryEntry/7960428545/CourseNode/92427926399928/Merkblatt_Literaturangaben_historische_Musikwissenschaft.pdf" rel="documentation"/>
+    <author>
+      <name>Jens Mittelbach</name>
+      <uri>http://www.slub-dresden.de</uri>
+      <email>jens.mittelbach@slub-dresden.de</email>
+    </author>
+    <category citation-format="note"/>
+    <category field="humanities"/>
+    <category field="generic-base"/>
```
Comment (rmzelle) [inline technische-universitat-dresden-historische-musikwissenschaft-note.csl]: Can you remove the "generic-base" field line? We typically only use it when we can't assign any other field, or if a style clearly is meant for a multidisciplinary audience (like APA).

File: technische-universitat-dresden-historische-musikwissenschaft-note.csl (hunk 1 above)
Comment (jmiba) [inline technische-universitat-dresden-historische-musikwissenschaft-note.csl]: I thought I might have to do that. So I will.

**Comment by rmzelle** [comment]
Thanks!

**Pull request merged**